server/sonar-server/src/main/java/org/sonar/server/authentication/ws/LoginAction.java

```diff
@@ -55,15 +54,15 @@
   private static final String LOGIN_ACTION = "login";
   public static final String LOGIN_URL = "/" + AUTHENTICATION_CONTROLLER + "/" + LOGIN_ACTION;
 
-  private final DbClient dbClient;
   private final CredentialsAuthenticator credentialsAuthenticator;
   private final JwtHttpHandler jwtHttpHandler;
   private final ThreadLocalUserSession threadLocalUserSession;
   private final AuthenticationEvent authenticationEvent;
+  private final UserSessionFactory userSessionFactory;
 
-  public LoginAction(DbClient dbClient, CredentialsAuthenticator credentialsAuthenticator, JwtHttpHandler jwtHttpHandler,
-    ThreadLocalUserSession threadLocalUserSession, AuthenticationEvent authenticationEvent) {
-    this.dbClient = dbClient;
+  public LoginAction(CredentialsAuthenticator credentialsAuthenticator, JwtHttpHandler jwtHttpHandler,
+    ThreadLocalUserSession threadLocalUserSession, AuthenticationEvent authenticationEvent, UserSessionFactory userSessionFactory) {
+    this.userSessionFactory = userSessionFactory;
```

Comment:
same comment as above:
usually, order of constructor arguments matches the order of properties and order they are assigned in the constructor